csv-data-management/info.json

```diff
@@ -75,6 +75,61 @@
           "description": "Number of rows you want to skip from top of CSV File. Note first row will be skipped even if it has column names",
           "tooltip":"Number of rows you want to skip from top of CSV File. Note first row will be skipped even if it has column names"
         },
+        {
+          "title": "Filter Dataset",
+          "type": "select",
+          "name": "filterInput",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "filtervalue": "",
+          "options": [
+            "Use Regex as Filter",
+            "Use 'isin' as Filter"
+          ],
+          "onchange" :{
+            "Use Regex as Filter" :[{
+              "title": "Filter on Column",
+              "type": "text",
+              "name": "filterColumnName",
+              "required": true,
+              "visible": true,
+              "editable": true,
+              "value": "" 
+
+            },
+            { 
+              "title": "Regex Filter",
+              "type": "text",
+              "name": "filter",
+              "required": true,
+              "visible": true,
+              "editable": true,
+              "value": ""
+            }],
+            "Use 'isin' as Filter":[{
+              "title": "Filter on Column",
+              "type": "text",
+              "name": "filterColumnName",
+              "required": true,
+              "visible": true,
+              "editable": true,
+              "value": "" 
+
+            },
+              {
+              "title": "Filter String",
```

Comment:
can we add a tooltip that says that user needs to specify a comma separated list of strings?
and a placeholder with an example